.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
```

Comment:
This file must no be on the project.

Reply:
I can remove it. But I suggest you think about using EditorConfig as is makes collaborating way more easier. I use it in my projects all the time. And I guess it's some kind of common sense.

Have a look here: http://editorconfig.org/